Code/Framework/AzCore/AzCore/Math/Quaternion.h

```diff
@@ -232,9 +240,11 @@ namespace AZ
         Vector3 GetEulerRadians() const;
 
         //! @param eulerRadians A vector containing component-wise rotation angles in radians.
+        //! @deprecated
```

Comment:
update: Please use O3DE_DEPRECATION_NOTICE and create a GHI to track this (see https://github.com/o3de/community/pull/141)